pyjobshop/ortools/solve.py

```diff
@@ -54,6 +57,8 @@ def solve(
     for key, value in params.items():
         setattr(cp_solver.parameters, key, value)
 
+    cp_solver.log_callback = print  # TODO how to enable this only in tests?
```

Comment:
This is blocking. This cannot be enabled outside of testing, because it will create duplicated log statements. But I cannot think of a nice way to have this only enabled in tests.